src/components/atoms/Ranges/LevelRange/LevelRange.styled.ts

```diff
@@ -0,0 +1,167 @@
+import styled from 'styled-components';
+import colors from 'styles/colors';
+
+export const StyledWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
```

Comment:
Why it has 100vh and 100% width ?

Reply:
My bad. I forgot to delete. I used these variables for centered components.